web/app/js/components/Sidebar.jsx

```diff
@@ -193,14 +193,14 @@ class Sidebar extends React.Component {
 
             <Menu.Item className="sidebar-menu-item" key="/tap">
               <PrefixedLink to="/tap">
-                <i className="fas fa-microscope" />
+                <Icon><i className="fas fa-microscope" /></Icon>
```

Comment:
this appears to indent Tap and Top more than the other menu elements?

![screen shot 2018-09-17 at 11 26 34 am](https://user-images.githubusercontent.com/236915/45642267-942cd300-ba6c-11e8-99b2-66f81ce14bb0.png)

Reply:
ah! good spot